From 7d190b49e8b2227398b173c2ca03160f153dd67c Mon Sep 17 00:00:00 2001
From: Enric Balletbo i Serra <eballetbo@gmail.com>
Date: Tue, 3 Apr 2012 19:34:17 +0200
Subject: [PATCH] OMAP3: igep00x0: fix typo in gpio request.

For 'nreset' pin the gpio pin requested is 'irq' which is incorrect. This
patch fixes following message.

   IGEP: Could not obtain gpio MCP251X NRESET

Signed-off-by: Enric Balletbo i Serra <eballetbo@gmail.com>
---
 arch/arm/mach-omap2/board-igep00x0.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-omap2/board-igep00x0.c b/arch/arm/mach-omap2/board-igep00x0.c
index aec61a8..c0a5402 100644
--- a/arch/arm/mach-omap2/board-igep00x0.c
+++ b/arch/arm/mach-omap2/board-igep00x0.c
@@ -441,7 +441,7 @@ void __init igep00x0_mcp251x_init(int bus_num, int cs, int irq, int nreset)
 	}
 
 	if (nreset) {
-		if ((gpio_request(irq, "MCP251X NRESET") == 0)
+		if ((gpio_request(nreset, "MCP251X NRESET") == 0)
 			&& (gpio_direction_output(nreset, 0) == 0))
 			gpio_export(nreset, 0);
 		else
-- 
1.7.1

